scripts/transact/mintLandsFromFile.ts

```diff
@@ -0,0 +1,68 @@
+import BN from 'bn.js';
+import fs from 'fs-extra';
+import {ethers, deployments} from 'hardhat';
+
+const networkName = 'rinkeby';
```

Comment:
This should be a param probably. If moving it to a param, can you add a small comment at the top on how the script should be called?

Reply:
done. thanks!